### DISCUSSION TITLE
1.30.2

### DISCUSSION THREAD

**Discussion by Koenkk** [Koenkk/zigbee2mqtt#16861] [General]
# Improvements
- [#5415](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5415) Improve Namron 4512737/4512738 reporting intervals (@owangen)
- [#5420](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5420) Add OTA support for Xiaomi `GZCGQ11LM` (@Otnow)
- [#651](https://github.com/Koenkk/zigbee-herdsman/pull/651) Improve debug logging for routing table/LQI request failure (@TomTonic)
- #16520 Expose min/max brightness for TuYa TS110E_1gang_1 (@Koenkk)
- [#657](https://github.com/Koenkk/zigbee-herdsman/pull/657) [#669](https://github.com/Koenkk/zigbee-herdsman/pull/669) [#673](https://github.com/Koenkk/zigbee-herdsman/pull/673) Improve EZSP integration (@kirovilya)
- [#456](https://github.com/zigbee2mqtt/hassio-zigbee2mqtt/issues/456) Support `backlight_mode` for TuYa TS0011 (@Koenkk)
- #16325 Support `do_not_disturb` and `color_power_on_behavior` for more TuYa lights (@Koenkk)
- #16535 Support `power_on_behavior` for Lidl HG06337, HG06338, HG06620 and HG06619 (@Koenkk)
- #15970 Support `backlight_mode` for TuYa TS0003_switch_module (@Koenkk)
- [#5449](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5449) Support OTA for Third Reality 3RSP02028BZ (@weihuan1111)
- [#5058](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5058) Support schedule for Xiaomi SRTS-A01 (@protyposis)
- [#255](https://github.com/Koenkk/zigbee-OTA/pull/255) Support OTA for Namron 4512700 (@UsynligAnd)
- [#5432](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5432) Improve TuYa TS0601_thermostat_1 integration (@Delorien84)
- [#5447](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5447) Enable Hue effects by default (@sjorge)
- [#5452](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5452) Support cooling for SMaBiT AV2010/32 (@sjorge)
- [#5454](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5454) Support full speed range for Mercator SSWF01G (@davidjb)
- [#661](https://github.com/Koenkk/zigbee-herdsman/pull/661) Improve EZSP integration (@kirovilya)
- [#258](https://github.com/Koenkk/zigbee-OTA/pull/258) Support OTA for Danalock V3-BTZBE (@Schluggi)
- [#5484](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5484) Set colortemp range for Calex 421792 (@JoyceKimberly)
- [#5486](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5486) Support electrical measurements for ROBB ROB_200-011-0 (@BoGu5)
- #16744 Add `entity_category` to `indicator_mode` and `led_enable` for Home Assistant discovery (@nattgris)
- [#5506](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5506) Expose `power` and `energy` for OWON WSP402 and WSP404 (@embak)
- [#5512](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5512) Support `battery` `get` for Ubisys H1 (@sjorge)
- [#5515](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5515) Improve Ubisys H1 `pi_heating_demand ` reporting (@sjorge)
- [#5516](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5516) Support electrical measurements for Third Reality 3RSP02028BZ (@llhappier)

# Fixes
- [#16454](https://github.com/Koenkk/zigbee2mqtt/issues/16454) Fix ROBB ROB_200-007-0 battery % multiplied by two (@Koenkk)
- #16485 Detect `_TYZB01_uqkphoed` as Lonsonho QS-Zigbee-S04-2C-LN (@Koenkk)
- [#5431](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5431) Fix Ajax Online ZB-CCT_Filament fingerprint (@cooper645)
- #16507 Fix electrical measurements not working for `_TZ3000_tgddllx4` (@Koenkk)
- [#5437](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5437) Fix Moes BRT-100-TRV time units (@tefracky)
- [#5446](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5446) Remove static `device_temperature` for Xiaomi CTP-R01 (@JJPro)
- #16467 Fix `occupancy` not reported for TuYa MIR-HE200-TY (@Koenkk)
- #16587 Correct some time units to prevent Home Assistant warnings (@Koenkk)
- [#5461](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5461) Fix TuYa TS0121_plug spamming electrical measurements (@stokket)
- #16398 Fix `Failed to call 'Bridge' 'start' (TypeError: Cannot read properties of undefined (reading 'ieeeAddr')` error (@Koenkk)
- #16479 Fix Home Assistant template warnings for covers (@Drafteed)
- [#5462](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5462) Fix Centralite 4257050-RZHAC power value (@bdd)
- #16615 Fix TuYa TS0601_gas_sensor_2 integration (@Koenkk)
- [#5456](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5465) Fix battery percentage of some Ikea devices (@sjorge)
- #16642 Detect `_TZE200_gd4rvykv` as TuYa TS0601_thermostat_3 (@Koenkk)
- [#664](https://github.com/Koenkk/zigbee-herdsman/pull/664) Fix Deconz adapter request timeout (@ercksen)
- #16605 Fix error when device is added when `device: []` in `configuration.yaml` (@zeroflag0)
- #16676 Detect `_TZ3000_f09j9qjb` as Zemismart TB25 (@Koenkk)
- [#5473](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5473) Fix TuYa TV02-Zigbee `current_heating_setpoint` access (@mattheworiordan)
- #15725 Fix no actions from IKEA E2001/E2002 with firmware 2.4.5 (@Koenkk)
- #16272  Fix Salus Controls PS600 not recognised (@mikeyyuen)
- [#5487](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5487) [#5489](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5489) [#5488](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5488) #16569 #16599 [#5479](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5479) [#16792](https://github.com/Koenkk/zigbee2mqtt/discussions/16792) Disable unsupported `power_on_behavior` for various devices (@aofc, @Koenkk, @11harveyj) 
- #15902 Fix TuYa TS110E_2gang_1 not turning on when controlled over Zigbee (@Koenkk)
- #16732 Fix configure of Legrand 067775/741811 failing (@Koenkk)
- [#5494](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5494) [#5495](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5495) [#5427](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5427) [#5511](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5511) Fix missing power source for various devices (@aofc, @SF6789, @llhappier )
-  [#5483](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5483) Filter erroneous values of SONOFF SNZB-02_EFEKTA (@MichelNDF)
- #16756 Fix `TypeError: Cannot create property 'message' on string 'APS TIMEOUT'` error (@yehzhang)
- [#5510](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5510) Fix Sinopé TH1300ZB `energy` value (@MattL0)
- [#5517](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5517) Reject incorrect battery reading (`255`) (@sjorge)
- [#5514](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5514) Fix Lidl 14149505L/14149506L color handling (@AndreKR)
- #16825 Fix min/max/change reporting interval being a string (@Koenkk)
- [#5529](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5529) Fix incorrect Third Reality 3RSP02028BZ electrical measurements (@weihuan1111)

# New supported devices
This release adds support for 44 devices:
- `067755` Legrand Wireless and batteryless 4 scenes control (@jiquem)
- `12502` Scan Products Zigbee 3.0 switch (@Wiese-TKW)
- `5120.2110` Iluminize ZigBee 3.0 dimming actuator mini 1x 230V (@widder17)
- `74746` LEDVANCE LEDVANCE 74746 Sylvania smart+ Zigbee dimmable edge-lit panel (@nkiest)
- `8719514301528` Philips Hue Filament White Ambiance ST72 B22 LED warm-to-cool (@mattkrins)
- `915005988401` Philips Hue Gradient light tube compact black (@narc792)
- `929003055701` Philips Hue White Ambiance Devote (@gohlas)
- `DM2550ZB` Sinopé Zigbee Adaptive phase smart dimmer (@proyleg)
- `E2112` IKEA Vindstyrka air quality and humidity sensor (@peterkappelt)
- `EB-E14-FLA-CCT` EssentielB E14 flame CCT light bulb (@Chartreusito)
- `EB-E27-A60-CCT-FC` EssentielB E27 A60 CCT filament clear light bulb (@Chartreusito)
- `EB-E27-A60-CCT` EssentielB E27 A60 CCT light bulb (@Chartreusito)
- `EB-E27-A60-RGBW` EssentielB E27 A60 RGBW light bulb (@Chartreusito)
- `EB-E27-G95-CCT-FV` EssentielB Filament vintage globe light bulb (@Chartreusito)
- `EB-E27-ST64-CCT-FV` EssentielB Filament vintage edison light bulb (@Chartreusito)
- `EB-GU10-MR16-CCT` EssentielB GU10 MR16 CCT light bulb (@Chartreusito)
- `EB-GU10-MR16-RGBW` EssentielB GU10 MR16 RGBW light bulb (@Chartreusito)
- `EB-SB-1B` EssentielB Smart button (@Chartreusito)
- `QBKG28LM` Xiaomi Aqara smart wall switch H1 Pro (no neutral, double rocker) (@pyatins)
- `RFPR-ZB-SH-EU` Bosch Wireless motion detector (@dan3o3)
- `SX885ZB` Salus Controls miniSmartPlug (@normanr)
- `T2105` IKEA STOFTMOLN ceiling/wall lamp 10 warm light dimmable (@schettada)
- `TI0001-pir` Livolo Zigbee motion Sensor (@mobot95)
- `TS0003_switch_module_2` TuYa 3 gang switch module (@landsmanv)
- `TS0049` TuYa Water valve (@JoppyFurr)
- `TS0224` TuYa Smart light & sound siren (@Koenkk)
- `TS0601_din_3` TuYa Zigbee DIN energy meter (@cserem)
- `TS0601_fan_switch` TuYa Fan switch (@isakharov)
- `VES-ZB-DIM-004` Vesternet Zigbee dimmer (@martyn-vesternet)
- `VES-ZB-HLD-017` Vesternet Zigbee high load switch (@martyn-vesternet)
- `VES-ZB-MOT-019` Vesternet Zigbee motor controller (@martyn-vesternet)
- `VES-ZB-REM-013` Vesternet Zigbee remote control - 12 button (@martyn-vesternet)
- `VES-ZB-SWI-005` Vesternet Zigbee switch (@martyn-vesternet)
- `VES-ZB-SWI-015` Vesternet Zigbee 2 channel switch (@martyn-vesternet)
- `VES-ZB-WAL-006` Vesternet Zigbee wall controller - 2 button (@martyn-vesternet)
- `VES-ZB-WAL-011` Vesternet Zigbee wall controller - 4 button (@martyn-vesternet)
- `VES-ZB-WAL-012` Vesternet Zigbee wall controller - 8 button (@martyn-vesternet)
- `WISZB-138` Develco Window sensor (@Frederik91)
- `YRD410-BLE` Yale Assure lock 2 (@corporategoth)
- `YRD420-BLE` Yale Assure lock 2 (@corporategoth)
- `YRD430-BLE` Yale Assure lock 2 (@corporategoth)
- `YRD450-BLE` Yale Assure lock 2 (@corporategoth)
- `ZL13100314` Linkind Ceiling light 28W, 3000 lm, Ø40CM CCT (@Ullulaki)
- `ZNQBKG31LM` Xiaomi Aqara E1 3 gang switch (with neutral) (@wilsonkhlam)

<hr /><em>This discussion was created from the release <a href='https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.30.2'>1.30.2</a>.</em>

**Comment by lux73** [comment]
updated from 1.30.0 without any issues

Thx for the steady Updates!
